packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -301,6 +306,8 @@ export class List extends BaseComponent<IListProps, IListState> implements IList
       pageElements.push(this._renderPage(pages[i]));
     }
 
+    // console.log(`Page elements ${pageElements.length}`);
```

Comment:
oh....I see a few other commented out console.logs. Guess they are left there for future testing?